x-pack/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/index.tsx

```diff
@@ -78,7 +82,35 @@ function getTabs(context, logStackframes) {
   ];
 }
 
-function DetailView({ errorGroup, urlParams, location }) {
+interface ErrorGroupData {
+  error: Error;
+  occurrencesCount: number;
+  groupId: string;
+}
+
+interface ErrorGroup {
+  status: string;
+  data: ErrorGroupData;
+  args: Array<unknown>;
+}
+
+interface UrlParams {
+  serviceName: string;
+  kuery: string;
+  detailTab: string;
+}
```

Comment:
This is already defined as `IUrlParams`:
https://github.com/elastic/kibana/blob/b3bd950400963293bc72e2b8dd0ec0bccfb1b644/x-pack/plugins/apm/public/store/urlParams.ts#L159-L175